dspace-spring-rest/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -757,14 +757,18 @@ public ResourceSupport findRel(HttpServletRequest request, HttpServletResponse r
 
         } else {
             if (resource.getEmbeddedResources().get(rel) == null) {
-                response.setStatus(HttpServletResponse.SC_NO_CONTENT);
+                if (model.equals("submissiondefinitions")) {
```

Comment:
this is not necessary as we want to deal with all the embedded resource collections in an uniform way, so it will be always 200  with an empty array